wiki/javascripts/download.js

```diff
@@ -0,0 +1,51 @@
+/* download.js script is used to update the jitpack releases of semantics
+ *  in gh-pages dynamically and to support the downloading functionality 
```

Comment:
Do we even need this file anymore?

Reply:
This file is no more needed. Hence removed this file